Simplenote/SPAppDelegate.m

```diff
@@ -96,8 +96,8 @@ - (void)setupAppCenter
     NSLog(@"Initializing AppCenter...");
     
     NSString *identifier = [SPCredentials appCenterIdentifier];
-    [MSAppCenter start:identifier withServices:@[[MSDistribute class]]];
-    [MSDistribute setEnabled:true];
+    [MSACAppCenter start:identifier withServices:@[[MSACDistribute class]]];
+    [MSACDistribute setEnabled:true];
```

Comment:
These were breaking the installable build so I updated them to the new API calls.